### DISCUSSION TITLE
CP routes being served over HTTP (not HTTPS) and breaking CMS

### DISCUSSION THREAD

**Discussion by twilliamsorel** [statamic/cms#10432] [Troubleshooting] [answered]
Hey all, 
I'm trying to host a Statamic site on the Digital Ocean App Platform and am running into some trouble. While the front end view of the website is working fine, the CMS is breaking due to certain assets being served as HTTP instead of HTTPS in /cp routes. This is happening, despite setting my .env vars: APP_URL and ASSET_URL to https. 

I know similar issues have happened in the past, but all of the existing related conversations that I can find are over 5 years old and no longer seem relevant, particularly to my situation.

Why might this be happening and how can I fix it? Any help here is greatly appreciated.

**Comment by edalzell** [comment]
Let's see your `sites.yaml`

**Reply by twilliamsorel** [reply]
Hm, I don't have one of those, and cant seem to find anything in the current docs or in the filesystem (sites.yaml didn't come with the install). Would you be able to point me to a resource to learn more about it? For reference, I'm using the latest version of statamic.

**Reply by edalzell** [reply]
should be in your `resources` folder. What version are you using?

**Reply by twilliamsorel** [reply]
Ah, found it; oops! Though, the trusted proxies solution worked, and I suspect addressed the core issue effectively.

**Reply by edalzell** [reply]
maybe, what was your site url set to?

**Reply by twilliamsorel** [reply]
the url was root: /

**Reply by edalzell** [reply]
right, that should be set to `'{{ config:app:url }}`

**Reply by twilliamsorel** [reply]
Gotcha, maybe that's the simpler solution. TY for the input -- I'll change that as well.

**Comment by sylvesterdamgaard** [comment, marked as answer]
Did you enable trust proxies? That solves this issue for most deployments in the cloud.

Laravel 11: https://laravel.com/docs/11.x/requests#configuring-trusted-proxies
Laravel 10: https://laravel.com/docs/10.x/requests#configuring-trusted-proxies

**Reply by twilliamsorel** [reply]
This worked. Thank you kindly, sir!
